### PR TITLE
Add H2 Migration Tool

### PR DESCRIPTION
**Pull request by ImalshaG**
This PR will add a migration script and the README file needed for the H2 DB upgrade from 1.4.199 to 2.1.210.

### CODE REVIEW

File: h2-migration-tool/migration.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+done
+
+rm -f h2-1.4.199.jar
+rm -f h2-2.1.210.jar
+
+echo
+echo "Database files are created at $dest_dir."
+echo
+echo "You can copy these files and replace the files inside the <IS-HOME>/repository/database folder."
+echo
```
Comment (piraveena) [inline h2-migration-tool/migration.sh:38]: Add new line

File: h2-migration-tool/migration.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
```
Comment (senthalan) [inline h2-migration-tool/migration.sh:1]: let's add licence header

**Review by piraveena — APPROVED** [review]

File: h2-migration-tool/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+   
+      Eg. /User/.wso2-updates/backup/wso2is-5.11.0-30de1a18-f60f-4bb7-8832-c84d7f244d85/repository/database
+   
+   2. Provide the path to a directory where the new db files will be created and stored
+   
+      Eg. < path>/new-databases
+   
+5. Copy the database files from the created location and replace the files inside the <IS-HOME>/repository/database directory.
+
+
```
Comment (chamathns) [inline h2-migration-tool/README.md:35]: redundant lines

Comment (ImalshaG) [reply h2-migration-tool/README.md:35]: Addressed by https://github.com/wso2/samples-is/pull/316.

**Pull request merged**